easybuild/easyconfigs/r/R/R-3.2.3-foss-2015b.eb

```diff
@@ -0,0 +1,400 @@
+name = 'R'
+version = '3.2.3'
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://cran.us.r-project.org/src/base/R-%(version_major)s']
+
+preconfigopts = 'BLAS_LIBS="$LIBBLAS" LAPACK_LIBS="$LIBLAPACK"'
+configopts = "--with-lapack --with-blas --with-pic --enable-threads --with-x=no --enable-R-shlib"
+# some recommended packages may fail in a parallel build (e.g. Matrix), and we're installing them anyway below
+configopts += " --with-recommended-packages=no"
+
+dependencies = [
+    ('libreadline', '6.3'),
+    ('ncurses', '5.9'),
+    ('libpng', '1.6.17'),  # for plotting in R
+    ('libjpeg-turbo', '1.4.1'),  # for plottting in R
+    ('Java', '1.8.0_72', '', True),  # Java bindings are built if Java is found, might as well provide it
+    ('Tcl', '8.6.4'),  # for tcltk
+    ('Tk', '8.6.4', '-no-X11'),  # for tcltk
+    ('cURL', '7.43.0'),  # for RCurl
+    ('libxml2', '2.9.2'),  # for XML
+#    ('GDAL', '2.0.1'),  # for rgdal
+]
+
+name_tmpl = '%(name)s_%(version)s.tar.gz'
+ext_options = {
+    'source_urls': [
+        'http://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'http://cran.r-project.org/src/contrib/',  # current version of packages
+        'http://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': name_tmpl,
+}
+
+
+# !! order of packages is important !!
+# packages updated on January 21st 2016
+exts_list = [
+    # default libraries, only here to sanity check their presence
+    'base',
+    'datasets',
+    'graphics',
+    'grDevices',
+    'grid',
+    'methods',
+    'splines',
+    'stats',
+    'stats4',
+    'tools',
+    'utils',
+    # non-standard libraries, should be specified with fixed versions!
+    #('Rmpi', '0.6-5', dict(ext_options.items() + [('patches', ['Rmpi-0.6-5_impi5.patch'])])),
```

Comment:
this patch is irrelevant if you're not building with Intel MPI, so just drop it?